dubbo-all/pom.xml

```diff
@@ -387,6 +387,13 @@
             <scope>compile</scope>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>org.apache.dubbo</groupId>
+            <artifactId>dubbo-serialization-googlePb</artifactId>
```

Comment:
let's call it 'dubbo-serialization-protobuf-json'

Reply:
done.